src/Features/LanguageServer/Protocol/Handler/Completion/CompletionResolveData.cs

```diff
@@ -4,9 +4,7 @@
 
 #nullable disable
```

Comment:
if it's not too impactful, could you remove this?

Reply:
It complains about the underlying `TextDocumentIdentifier` below

Reply:
I'm an idiot, I removed the TextDocumentIdentifier which was outside of our control and misremembered. YOu are 100% right